amundsen_application/static/js/components/common/Tags/TagsList/index.tsx

```diff
@@ -77,10 +69,10 @@ const ShortTagsList: React.FC<TagsListProps> = ({
   const hasPopularTags = popularTags.length > 0;
   return (
     <div className="short-tag-list">
-      {!hasCuratedTags && hasPopularTags && (
-        <TagsListTitle titleText={POPULAR_TAGS_TITLE} />
-      )}
-      {hasCuratedTags && <TagsListTitle titleText={CURATED_TAGS_TITLE} />}
+      <h2 className="tag-list-title" id="browse-header">
```

Comment:
nit: Do we want the `h2` to still render even if the content may be empty if `!hasCuratedTags && !hasPopularTags`?

Reply:
Oh, that's a good point.
I was assuming they will always show something, but that's a legit case!
I will update it, thanks!